python/tk_multi_workfiles/ui/entity_tree_form.py

```diff
@@ -3,7 +3,7 @@
 ################################################################################
 ## Form generated from reading UI file 'entity_tree_form.ui'
 ##
-## Created by: Qt User Interface Compiler version 5.15.2
+## Created by: Qt User Interface Compiler version 5.15.16
```

Comment:
Should we freeze a specific version to the "patch" version?

Reply:
5.15.16 was the version I used to generated the UI sources. It doesn't make any difference in this PR but in the future, I'd suggest we should generate files with PySide versions we support.